package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rsksmart/rsk-precompiled-abis",
-  "version": "5.0.0-HOP",
+  "version": "6.0.0",
```

Comment:
Sorry Jere, I meant the version should be 6.0.0-HOP. It's [versionNumber-releaseName]. Once we know the name of the next release we can change it.

Sorry for the mix-up 😬

Reply:
Done!